prov/opx/src/fi_opx_reliability.c

```diff
@@ -2354,6 +2402,48 @@ uint8_t fi_opx_reliability_service_init (struct fi_opx_reliability_service * ser
 
 	service->usec_next = fi_opx_timer_next_event_usec(&service->tx.timer, &service->tx.timestamp, service->usec_max);
 
+	/*
+	* Initialize send ping flag(s)
+	*
+	* ONLOAD only
+	*/
+	service->tx.congested_flag = 0;
+
+	/*
+	 * Maximum number of reliability pings per timer in congested/uncongested scenarios
+	 *
+	 * OFFLOAD and ONLOAD
+	 */
+	int max_uncongested_pings;
+	if(fi_param_get_int(fi_opx_global.prov, "reliability_max_uncongested_pings", &max_uncongested_pings) == FI_SUCCESS) {
+		if (max_uncongested_pings < OPX_RELIABILITY_MAX_UNCONGESTED_PINGS_MIN || max_uncongested_pings > OPX_RELIABILITY_MAX_UNCONGESTED_PINGS_MAX) {
```

Comment:
Try break up the super long line for better readability. somthing like:
```
        if (max_.... < OPX_.... ||
            max_.... > OPA_....) {
```

Reply:
Will address in a cleanup commit.